python/pylibcudf/pylibcudf/io/experimental/hybrid_scan.pyi

```diff
@@ -0,0 +1,130 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION.
+# SPDX-License-Identifier: Apache-2.0
+
+from enum import IntEnum
+
+from rmm.pylibrmm.device_buffer import DeviceBuffer
+from rmm.pylibrmm.memory_resource import DeviceMemoryResource
+from rmm.pylibrmm.stream import Stream
+
+from pylibcudf.column import Column
+from pylibcudf.io.parquet import ParquetReaderOptions
+from pylibcudf.io.types import TableWithMetadata
+
+class UseDataPageMask(IntEnum):
+    YES: int
+    NO: int
+
+class ByteRangeInfo:
```

Comment:
Interesting to see we didn't already have bindings for this

Reply:
Good point. I moved this over to io.text since that's where this type is defined in libcudf. We just never had a corresponding Python type before.